README.md

```diff
@@ -12,3 +12,170 @@ the platform.
+is computed on dates when The Browser has a major release.
+
+#### Rationale
+
+When browsers move too slowly, it holds back the platform. When browsers move
+too quickly, they risk “leaving other browsers behind”. Steady growth is good;
+wild variation is bad.
+
+### Failure to Ship
+
+#### Definition
+
+The Failure to Ship metric counts APIs that The Browser does not provide, but
+all other browsers provide after the Grace Period.
+
+#### Rationale
+
+Failing to ship an API that other major vendors provide requires web developers
+to use special code paths to remain interoperable. Smaller values are good;
+larger values are bad.
+
+#### Definition
+
+The Aggressive Removal metric counts APIs that The Browser once provided, but no
+longer does, and all other browsers provide after the Grace Period.
+
+#### Rationale
+
+Removing an API from only one browser risks breaking existing sites that
+(reasonably) assume that all browsers support the API. Smaller values are good;
+larger values are bad.
+
+#### Definition
+
+The Browser-Specific metric counts APIs that The Browser provides, but all other
+browsers do not provide after the Grace Period.
+
+#### Rationale
+
+Adding APIs that are provided by only one browser makes that browser more and
+more like its own platform (rather than an implementation of a common web
+platform). Smaller values are good; larger values are bad.
+
+## Contributing
+
+Want to contribute to Web API Confluence? Great!
+
+### Filing issues and contributing code
+
+Please use Github’s issue tracker and pull request features.
+
+### Running locally
+
+1. Before cloning this repository you
+   should [install git lfs](https://git-lfs.github.com/). We use Git LFS to
+   check in database snapshots. This improves the local debugging experience (no
+   need to hit remote APIs to access data locally) and improves production
+   deployment (snapshots bootstrap local caches).
+
+2. Clone this repository.
+
+3. Use `npm run serve` to launch a local instance of the service. This will look
+   for `.local/credentials.json`, and when it doesn’t find it, use the local
+   snapshot of data, rather than attempting to connect to a data backend. The
+   last data source to come up in the release-to-API associations (usually takes
+   about a minute).
+
+4. Hack away! `npm run serve` uses `webpack --watch` to observe local
+   changes. Making changes to server code will require a service restart, but
+   client-side changes will be reflected soon after they are saved.
+
+### Collecting data
+
+**NOTE**: The current data collection process requires
+a [BrowserStack](https://www.browserstack.com/)
+account, [Cloud Datastore](https://cloud.google.com/datastore/) credentials, two
+separate `git clone`s, and a whole lot of RAM. We hope to streamline and
+simplify this process soon. If you have all the prerequisites, read on…
+
+1. Clone [mdittmer/web-apis](https://github.com/mdittmer/web-apis) and follow
```

Comment:
Is there a plan to move this into the GoogleChrome organization at some point?

Reply:
Yeah, I guess. It depends on how high priority it is to make it *easy* (not just *possible*) for third parties to reproduce results.